tests/nat/mcp/test_mcp_auth_provider.py

```
@@ -787,102 +647,93 @@ async def test_discover_and_register_without_registration_endpoint(self, mock_co
 
                 # Verify the flow was called
                 mock_flow.assert_called_once()
-                # Check if the call was made with keyword arguments
-                if mock_flow.call_args and mock_flow.call_args.kwargs:
-                    auth_request = mock_flow.call_args.kwargs.get('auth_request')
-                    if auth_request:
-                        assert auth_request.user_id == "test_user"
 
     async def test_authenticate_retry_after_401_clears_auth_code_provider(self,
                                                                           mock_config_with_credentials,
                                                                           mock_endpoints):  # noqa: E501
```

Ruff (0.13.1)

Unused `noqa` directive (non-enabled: `E501`)

Remove unused `noqa` directive

(RUF100)